README.md

```diff
@@ -11,8 +11,8 @@ The issue bot helps by managing [Waiting for Triage](waiting-for-triage) and mon
 The issue bot labels issues and pull requests as waiting for triage.
 An issue or pull request is deemed as waiting for triage when all of the following is true:
 * It is open
-* It wasn’t opened by a collaborator
-* It isn’t assigned to a milestone
+* It was not opened by a collaborator
+* It is not assigned to a milestone
```

Comment:
What prompted the change here? "wasn't" and "isn't" seem fine to me.

Reply:
Not much besides I thought we’d use the formal way. As far as I know we use that style in the documentation and I thought this would apply to the `README` as well.

Reply:
I don't mind the slightly more informal style of "wasn't" and "isn't" in a README. That said, I don't mind the more formal style either. Happy to change, but can we do so consistently please? There are some other occurrences of "isn't" on the lines below.

Reply:
Thanks for the feedback. I've pushed an update